src/hotspot/share/opto/callnode.hpp

```diff
@@ -329,14 +329,14 @@ class SafePointNode : public MultiNode {
                 // A plain safepoint advertises no memory effects (NULL):
                 const TypePtr* adr_type = NULL)
     : MultiNode( edges ),
-      _oop_map(NULL),
       _jvms(jvms),
       _adr_type(adr_type)
   {
     init_class_id(Class_SafePoint);
   }
 
-  OopMap*         _oop_map;   // Array of OopMap info (8-bit char) for GC
+  // There is no OopMap field, the oop map is set after matching in
+  // OopFlow::compute_reach on the MachSafePointNode. (See buildOopMap.cpp)
```

Comment:
I don't think we need this comment. I think it would bitrot eventually. It seems the code to construct `OopMap` is easily discoverable already.